api/docs/API.doxy

```diff
@@ -239,7 +239,6 @@ DOT_IMAGE_FORMAT       = png
 DOT_PATH               =
 DOTFILE_DIRS           =
 MAX_DOT_GRAPH_DEPTH    = 1000
-DOT_TRANSPARENT        = NO
```

Comment:
I suppose we removed this because setting it to NO is a no-op since that is the default?

Reply:
Yes, the default value for DOT_TRANSPARENT was NO. See https://github.com/doxygen/doxygen/blob/a44f6a7e5caf1fbf5fe271c99046a51b315d5f56/src/config.xml#L3957

Reply:
Ok.